### ISSUE TITLE
clicking through cutscenes has weird text-flicker/could be more responsive

### ISSUE THREAD

**Issue by increpare** [TerryCavanagh/diceydungeons.com#1155]
from chat:

```
Oh, does it behave weirdly when you try to click quickly through a cutscene?

I feel like the text flickers sometimes, and it generally doesn't behave as elegantly as one might like)
```

**Comment by TerryCavanagh** [comment]
Should actually be an easy fix, fix this one before launch terryyyyy

**Comment by TerryCavanagh** [comment]
Hey, I finally fixed this one! Just in time for ~launch~v1.8!
